virt/getting_started/virt-web-console-overview.adoc

```diff
@@ -979,7 +978,7 @@ You create snapshots and restore virtual machines from snapshots on the *Snapsho
 
 Click the snapshot name to edit the labels or annotations.
 
-Click the actions menu {kebab} beside a snapshot to select *Restore* or *Delete*.
+Click the actions menu {kebab} beside a snapshot to select *Clone*, *Restore*, or *Delete*.
```

Comment:
*Clone* is there at the moment, but UI team needs to change it, because it's a bug

Reply:
Yes, I've modified the step to show the new options.

Reply:
@yfrimanm, In the updated design doc, the kebab options shown below Create VirtualMachine in the Snapshots table do not seem accurate. I've mentioned only these options where we describe them in [Snapshot table description in the Snapshots tab](https://71208--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/getting_started/virt-web-console-overview#virtualmachine-details-snapshots_virt-web-console-overview): Create VirtualMachine, Restore, and Delete. Is this correct?